### PR TITLE
ramips: enable Wi-Fi LED support for Afoundry EW1200 and Zbtlink ZBT-WE1326 

### PR DESCRIPTION
**Pull request by DragonBluep**
Add LED properties to PCIe node to enable Wi-Fi LED support.

Fixes:
https://github.com/openwrt/mt76/issues/718
https://github.com/openwrt/openwrt/issues/11530

### CODE REVIEW

**Comment by DragonBluep** [comment]
@dfateyev @EddieTH It would be great if you could add a `Tested-by` tag with your name and E-mail.

**Review by EddieTH — APPROVED** [review]
<code class="notranslate">Tested-by</code> @EddieTH on Afoundry EW1200

**Comment by dfateyev** [comment]
@DragonBluep as for ZBT WE1326 LED configuration: he have already covered it in https://github.com/openwrt/openwrt/pull/11220, right?
I believe, the PR should address Afoundry EW1200 changes only.

**Comment by DragonBluep** [comment]
@dfateyev #11220 is blocked by kernel 5.15 bump. And we can repair the LED here in advance. After this PR is merged, I will update and delete the LED commit in #11220.

**Comment by dfateyev** [comment]
@DragonBluep beside LED configuration fixes, ZBT WE1326 requires other fixes mentioned in #11220. Splitting existing PRs can only distract participants attention IMO. But if you feel it's better — then let's move the LED part into this PR.
For all other fixes in WE1326, we will need to wait for #11220 merge, anyway.

**Pull request merged**
